k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1173,6 +1194,12 @@ private void processSourceMessage(MessageViewInfo messageViewInfo) {
                     break;
                 }
             }
+        } catch (IOException ioe) {
```

Comment:
This is too broad. It should wrap only the individual calls that we reasonable expect to throw it (I think that's just Forward and Report ?). That way if the underlying API for the other methods is changed the developer has to think how to handle an IOE.